src/main/java/uk/gov/companieshouse/api/accounts/exception/handler/GlobalExceptionHandler.java

```diff
@@ -35,6 +37,16 @@
     @Value("${invalid.value}")
     public String invalidValue;
 
+    @ExceptionHandler(value = {Exception.class})
+    @ResponseStatus(value = HttpStatus.INTERNAL_SERVER_ERROR)
```

Comment:
Do we need this as well as the return? I believe you can make the method `void` or remove this annotation